src/containers/DAO/pages/DAOList/DAOTable/DAOTableRow.tsx

```diff
@@ -52,7 +52,7 @@ const DAOTableRow: React.FC<DAOTableRowProps> = ({ data }) => {
 		if (totalUsd) {
 			add({ zna: data.zna, total: totalUsd });
 		}
-	}, [totalUsd]);
+	}, [add, data.zna, totalUsd]);
```

Comment:
these missing dependents were causing the stat widget total to be incorrect on DAO homepage